src/components/SlateEditor/plugins/rephrase/Rephrase.tsx

```diff
@@ -92,7 +95,7 @@ export const Rephrase = ({ attributes, editor, element, children }: Props) => {
   };
 
   return (
-    <DialogRoot defaultOpen onExitComplete={onClose}>
+    <DialogRoot defaultOpen closeOnEscape closeOnInteractOutside onExitComplete={onClose}>
```

Comment:
Er ikke begge disse default-true?

Reply:
De gjorde så en bug var mindre gjenskapende så jeg mente dette var løsningen. Viser seg at du nesten må ha satt `at` parameteren for å fjerne/unwrape noder 🙃